packages/uswds-core/src/styles/settings/_settings-components.scss

```diff
@@ -110,6 +110,18 @@ $theme-identifier-max-width: "desktop" !default;
 $theme-identifier-primary-link-color: default !default;
 $theme-identifier-secondary-link-color: "base-light" !default;
 
+// In-Page Navigation
+$theme-in-page-nav-background-color: "default" !default;
+$theme-in-page-nav-background-padding: 2 !default;
+$theme-in-page-nav-background-radius: "lg" !default;
+$theme-in-page-nav-bar-color: "default" !default;
+$theme-in-page-nav-bar-width: "05" !default;
```

Comment:
Flagging that large values here will cause the bar to overlap list item text. 
`$theme-in-page-nav-bar-width: 3`:

![image](https://user-images.githubusercontent.com/93996430/199755651-16084da4-0fb2-495d-ab18-3c35044ab789.png)

Reply:
Honestly, I wonder if we should just fix the bar width for our release version

Reply:
On second though, I'm going to leave as-is and leave it up to the teams to use a working value when they customize...